source/security/threatmonitoring/non-ukfast-install.md

```diff
@@ -0,0 +1,128 @@
+# Installing Threat Monitoring on a non-UKFast server
+
+```eval_rst
+.. warning::
+   This documentation is for setting on a non-UKFast hosted server with Threat Monitoring. Instructions on how to install Threat Monitoring and Response on a UKFast hosted server can be found in our [Installing Threat Monitoring on a UKFast hosted server](/security/threat-monitoring/ukfast-hosted-install) documentation.
+```
+
+Threat Monitoring can also be installed on servers that are not hosted with UKFast, allowing you to secure and protect all parts of your IT infrastructure via one centralised threat defence platform.
+
+To begin, log into MyUKFast and visit the Threat Monitoring new agent configuration page https://my.ukfast.co.uk/threat-monitoring/configuration then select `Non UKFast` from the `Hosting Type` drop-down option.
+
+![setup-type](files/setup-type.png)
+
+
+To install UKFast Threat Monitoring on a non-UKFast hosted server, follow these steps:
+
+**[1. Add an API token](#add-api-token)** to link your servers to Threat Monitoring
+
+**[2. Setup firewall](#setup-firewall)** your firewall to allow communication
+
+**[3. Configure and install](#configure-and-install)** threat monitoring onto your server.
+
+**[4. Setup email contacts](#setup-email-contacts)** to receive email alerts. 
+
+
+
+```eval_rst
+.. warning::
+   Threat Monitoring only supports a specific set of operating systems and configurations. Please make sure your server meets the minimum required specification and required requirements as defined in our [System Requirements](/security/threat-monitoring/system-requirements) documentation.
```

Comment:
Check this about hyperlinks in `eval_rst` blocks: https://github.com/ukfast/docs.ukfast.co.uk#hyperlinks